partners_sponsors.md

```diff
@@ -0,0 +1,10 @@
+# Partners and sponsors
+
+Here is a list of Yunohost partners ans sponsors:
+- [GLOBENET](http://www.globenet.org): activist association, at the service of freedom of expression, offering internet services.
+- [TETANEUTRAL-NET](https://tetaneutral.net/): associative Internet access provider currently operating a radio network in Toulouse and its surroundings and a hoster.
+- [GITOYEN](https://gitoyen.net): association bringing together several companies and associations acting as a provider of hosting infrastructure and Internet access.
+- [LDN-NET](https://ldn-fai.net/) : association for the defense of a free, neutral and decentralized Internet whose main means of action is to be an Internet access provider associative and local.
+- [NBS System](https://www.nbs-system.com/): company specialized in hosting, securing Clouds, outsourcing (Information Systems, SaaS Applications, Web Platforms) and managed services.
```

Comment:
Uuuuuh, not sure to understand how this company is our partner o.O

Reply:
You speak about `NBS System` ? It's the company where IsAAAc is working....

Reply:
Alrighty, sorry :sweat_smile: